private/pkg/protosource/option_extension_descriptor.go

```diff
@@ -127,7 +127,7 @@ func isDescendantPath(descendant, ancestor []int32) bool {
 	return true
 }
 
-func (o *optionExtensionDescriptor) Range(fn func(protoreflect.FieldDescriptor, protoreflect.Value) bool) {
+func (o *optionExtensionDescriptor) ForEachFieldDescriptor(fn func(protoreflect.FieldDescriptor, protoreflect.Value) bool) {
```

Comment:
This name doesn't seem any clearer than `Range` to me. I think `RangeOptions` or maybe `RangePresentOptions` is better, since it more clearly indicates that you are ranging over _option_ fields.

Reply:
how does `ForEachPresentOption` sound?

Reply:
yep thats what I got to as well, see below